### ISSUE TITLE
Header

### ISSUE THREAD

**Issue by akhil-ambady** [vmudigal/ngx-audio-player#49]
Thanks for developing this component.

I'm using your ngx-audio-player for streaming audio from URL.
The player is working fine. I need to know how to add authorized token to header for the URL.
This is to stream the audio file which is the response from an API which is authorized.

So it would be great if you address this issue.

**Comment by vmudigal** [comment]
Hi Akhil,

Do you have a solution for this? If so, send me the pull request.

It would also be good if you could share the exact steps that you are
trying to do with example.

Thanks.

Best Regards,
Vijayendra

**Comment by akhil-ambady** [comment]
No, I don't have a solution for this. I would like to know whether we can add an authorized token with header.

Thanks.

Best Regards, 
Akhil A Ambady

**Comment by kennyhitachi** [comment]
Just checking if there is any update on this issue. Or we have to build it referencing https://auth0.com/blog/building-an-audio-player-app-with-angular-and-rxjs/

**Comment by Eternal21** [comment]
I wasn't able to find any solutions for this.  Looks like the only option at this point, is downloading the file using your own API requests, and then pointing the player to the downloaded file.
